### PR TITLE
Email Draft Mailinglist coreos-users

### PR DESCRIPTION
**Pull request by zeisss**

### CODE REVIEW

File: docs/mailinglist-email-rc1.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+We would like to get some feedback on one of the tools we started developing last month: Inago.
+
+At Giant Swarm we are using fleet to schedule our workload onto our cluster. One
+of the patterns that emerged here, is what we call unit-chains. A group of units
+that must run on the same machine and is chained with Requires/After.
+
+We have developed two internal tools, which interact with fleet directly and operate on 
+those chains. Since we are currently [open sourcing](1) a bunch of tools, we believe
+a general tool to manage those unit-chains might be useful for others too. Since our
+two existing tools are too specialized, we started working on a more general approach:
```
Comment (JosephSalisbury) [inline docs/mailinglist-email-rc1.md]: We don't need a colon at the end here.

File: docs/mailinglist-email-rc1.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+With Inago you define a folder with unit files and similar to fleetctl you can
+submit, start, stop, destroy slices of this folder. This allows you to define
```
Comment (JosephSalisbury) [inline docs/mailinglist-email-rc1.md]: "submit, start, stop, and destroy slices"

File: docs/mailinglist-email-rc1.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+an app once, and start multiple copies for high availability.
+
+Another feature we added is update. Update is inspired by Kubernetes Rolling Update.
```
Comment (JosephSalisbury) [inline docs/mailinglist-email-rc1.md]: Kubernetes rolling update

File: docs/mailinglist-email-rc1.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+You provide a max-growth and min-alive parameter, and Inago dynamically creates new
+slices and shutsdown the old ones, to replace all slices with the new version.
```
Comment (JosephSalisbury) [inline docs/mailinglist-email-rc1.md]: shuts down

File: docs/mailinglist-email-rc1.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+You can find the code on GitHub[3] and a prebuild binary at [2]. Feel free to play around
```
Comment (JosephSalisbury) [inline docs/mailinglist-email-rc1.md]: prebuilt

File: docs/mailinglist-email-rc1.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+and give us feedback in the github issues.
```
Comment (JosephSalisbury) [inline docs/mailinglist-email-rc1.md]: GitHub issues

**Comment by JosephSalisbury** [comment]
:gb:

**Comment by puja108** [comment]
RFR @giantswarm/cena 

@JosephSalisbury's comments already fixed

**Comment by JosephSalisbury** [comment]
LGTM

**Comment by puja108** [comment]
send out, ok if I close here?

**Comment by JosephSalisbury** [comment]
please

**Pull request closed without merging**